contracts/SignerBondsEscrow.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity <0.9.0;
+
+import "./RiskManagerV1.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/// @title SignerBondsEscrow
+/// @notice Process incoming signer bonds by putting them into escrow the
+///         governance can withdraw from.
+contract SignerBondsEscrow is ISignerBondsProcessor, Ownable {
+    /// @notice Processes the signer bonds.
+    /// @dev Adds incoming bonds to the overall contract balance.
+    function processSignerBonds() external payable override {}
+
+    /// @notice Withdraws collected bonds to the given target address.
+    /// @dev Can be called by the governance only.
+    /// @param target Arbitrary target address chosen by the governance.
```

Comment:
... that will be responsible for swapping ETH and depositing collateral to the coverage pool.

Reply:
See https://github.com/keep-network/coverage-pools/pull/39/commits/2c485c3441ff5c1a9283bb2fb9b8114a8fd0f899